src/field_effect.c

```diff
@@ -2036,7 +2043,8 @@ static bool8 LavaridgeGymB1FWarpEffect_Rise(struct Task *task, struct ObjectEven
             {
                 task->data[3]++;
             }
-        } else
+        }
+        else
```

Comment:
As above

Reply:
Merged upcoming which includes the pret merge that fixes this